lib/pharos/config_schema.rb

```diff
@@ -27,6 +27,7 @@ def self.messages
               optional(:user).filled
               optional(:ssh_key_path).filled
               optional(:container_runtime).filled(included_in?: ['docker', 'cri-o'])
+              optional(:master_taint).filled(:bool?)
```

Comment:
Could this be more generic `taints` array.. if user sets it empty then master is not tainted?

Reply:
See also previous discussion... IMO the master taint is a special case.

Reply:
Ok, but I think that `master_taint` is not generic enough as a host level option. It feels really weird tome if user can define this for hosts that have role=worker... and what happens when we introduce `taints` array?

Reply:
What would the taints array look like?

```yaml
hosts:
  - address: 192.0.2.1
    taints:
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
```

This will then need to (conditionally) disable the kubeadm-managed taints, and then patch the remaining taints into the node using the `LabelNode` phase.